mail_footer_notified_partners/__openerp__.py

```diff
@@ -0,0 +1,19 @@
+# -*- coding: utf-8 -*-
+# © 2016 ACSONE SA/NV <https://acsone.eu>
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+{
+    'name': "Notified partners in mail footer",
+    'summary': """
```

Comment:
I didn't find this naming convention into https://github.com/OCA/maintainer-tools. Could you point me where you found this?